calendar/calendar-bundles/resources/calendar_fr_FR.properties

```diff
@@ -477,7 +477,7 @@ prefs_low_priority=Priorit\u00e9 basse
 prefs_move_up=D\u00e9placer vers le haut
 prefs_move_down=D\u00e9placer vers le bas
 date_format=dd MMMMM yyyy
-date_link_format=aaaa-MM-jj
+date_link_format=yyyy-MM-dd
```

Comment:
I found this in 2 other places. The text usage in jobscheduler is _probably_ fine (I don't know French so don't know if this is the how they refer the the format) but the rwiki one should probably also be updated.

```
calendar/calendar-bundles/resources/calendar_fr_FR.properties
480:date_link_format=aaaa-MM-jj

jobscheduler/scheduler-test-component-shared/src/bundle/org/sakaiproject/component/app/scheduler/jobs/test/Messages_fr_FR.properties
22:error.startDate.format=la date de d\u00e9but doit \u00eatre de la forme 'aaaa-MM-jj' (par exemple\: 2010-11-01)

rwiki/rwiki-impl/impl/src/bundle/uk/ac/cam/caret/sakai/rwiki/component/bundle/Messages_fr_FR.properties
55:RecentChangesMacro.6=aaaa-MM-jj
```

Reply:
RecentChangesMacro.6 is not using rb.getFormattedMessage

final String openDateErrorDescription = rb.getFormattedMessage("java.alert.opendatedescription",
								calEvent.getField(CalendarUtil.NEW_ASSIGNMENT_OPEN_DATE_ANNOUNCED));

SimpleDateFormat format = new SimpleDateFormat(Messages.getString("RecentChangesMacro.6"), new ResourceLoader().getLocale()); //$NON-NLS-1$